index.html

```diff
@@ -835,7 +835,9 @@ <h2>Contact Info</h2>
                                         <div class="widget-content">
                                             <a href="tel:1800-574-9687" class="contact-number">+1 647-648-2979</a>
                                             <ul>
-                                                <li>10 capreol capreol court, Toronto</li>
+                                                <li><a href="https://goo.gl/maps/cVfifCcR3DCtpKju6" target="_blank">10
+                                                        capreol capreol
```

Comment:
Hi @SuneetaDevi , there is still double capreol capreol. 
AND The address should be exactly: 10 Capreol Court, Toronto, M5V 4B3